src/build-scripts/build_ninja.bash

```diff
@@ -14,8 +14,18 @@ NINJA_VERSION=${NINJA_VERSION:=1.10.2}
 NINJA_BRANCH=${NINJA_BRANCH:=v${NINJA_VERSION}}
 NINJA_INSTALL_DIR=${NINJA_INSTALL_DIR:=${LOCAL_DEPS_DIR}/dist/bin}
 
-if [ ! -f $DOWNLOADS_DIR/ninja-${NINJA_VERSION}.zip ]; then
-    curl --location ${NINJA_REPO}/archive/${NINJA_BRANCH}.tar.gz -o $DOWNLOADS_DIR/ninja-${NINJA_BRANCH}.tar.gz
+# Check if the directories exist, create them if not
+if [ ! -d "$LOCAL_DEPS_DIR" ]; then
+    mkdir -p "$LOCAL_DEPS_DIR"
+fi
```

Comment:
I think you don't need the `if` here: just the `mkdir -p` will do it, and the -p flag makes it not be an error if the directory already exists.

Reply:
I think Lee is right. The `mkdir -p "$DOWNLOADS_DIR"` should do the whole job, no?

Reply:
Yes, it was to able create the whole download directory.